src/test/java/edu/unc/lib/boxc/migration/cdm/services/SourceFilesSummaryServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.AbstractOutputTest;
+import edu.unc.lib.boxc.migration.cdm.model.MigrationProject;
+import edu.unc.lib.boxc.migration.cdm.model.SourceFilesInfo;
+import edu.unc.lib.boxc.migration.cdm.options.Verbosity;
+import edu.unc.lib.boxc.migration.cdm.test.CdmEnvironmentHelper;
+import edu.unc.lib.boxc.migration.cdm.test.SipServiceHelper;
+import edu.unc.lib.boxc.migration.cdm.util.ProjectPropertiesSerialization;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.time.Instant;
+
+public class SourceFilesSummaryServiceTest extends AbstractOutputTest {
+    private static final String PROJECT_NAME = "proj";
+    private static final String USERNAME = "migr_user";
+
+    @TempDir
+    public Path tmpFolder;
+
+    private MigrationProject project;
+    private SipServiceHelper testHelper;
+    private SourceFilesSummaryService summaryService;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        project = MigrationProjectFactory.createMigrationProject(
+                tmpFolder, PROJECT_NAME, null, USERNAME, CdmEnvironmentHelper.DEFAULT_ENV_ID);
+
+        testHelper = new SipServiceHelper(project, tmpFolder);
+        summaryService = new SourceFilesSummaryService();
+        summaryService.setProject(project);
+    }
+
+    @Test
+    public void summaryOutputTest() throws Exception {
+        testHelper.indexExportData("mini_gilmer");
+        Path path1 = testHelper.addSourceFile("25.txt");
+        writeCsv(mappingBody("25,," + path1 +","));
+
+        summaryService.summary(0, Verbosity.NORMAL);
+
+        assertOutputMatches(".*New Files Mapped: +1.*");
+        assertOutputMatches(".*Total Files Mapped: +1.*");
+        assertOutputMatches(".*Total Files in Project: +3.*");
+    }
+
```

Comment:
Some extra cases to test for, that the count is correct when:
* there are duplicate entries,
* there are entries with ids not in the index
* nothing is mapped
* there are entries with no path mapped
* everything is mapped (we want to know that it works for more than one mapped file)